mail_channel_notify_tag/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Mail Channel Notify Tag",
+    "summary": "Module to notify the channels when they are mentioned in the chatter",
+    "author": "Vauxoo",
+    "website": "https://www.vauxoo.com",
+    "license": "OPL-1",
+    "category": "Productivity/Discuss",
+    "version": "15.0.1.0.0",
+    "depends": [
+        "mail",
+    ],
+    "data": [
+        "views/mail_templates.xml",
+    ],
+    "demo": [],
+    "assets": {
+        "web.assets_backend": [
+            "mail_channel_notify_tag/static/src/models/composer_view/composer_view.esm.js",
+        ],
+        "mail.assets_discuss_public": [
+            "mail_channel_notify_tag/static/src/models/composer_view/composer_view.esm.js",
```

Comment:
Is it expected having this duplicated?

Reply:
Yes, composer view is loaded in the both assets [here](https://github.com/odoo/odoo/blob/9d5b57596836e63eb4d127151bf10dfa79186fa5/addons/mail/__manifest__.py#L83)